stablehlo/testdata/expm1_shape_float16_20_20.mlir

```diff
@@ -1,4 +1,4 @@
-// RUN-DISABLED: stablehlo-translate --deserialize %s.0_9_0.bc | stablehlo-opt -inline | stablehlo-translate --interpret
+// RUN-DISABLED(#1278): stablehlo-translate --deserialize %s.0_9_0.bc | stablehlo-opt -inline | stablehlo-translate --interpret
```

Comment:
I plan to add this in bulk in #1418. You may revert this file and the file below.